File: akka-scala-scheduler/src/main/scala/com/example/ApplicationMain.scala
package com.example

import java.util.Date

import akka.actor.ActorSystem

object ApplicationMain extends App {
  val system = ActorSystem("SchedulerActorSystem")
  val scheduleActor = system.actorOf(ScheduleActor.props, "ScheduleActor")
  1 to 1000 foreach { id => scheduleActor ! ScheduleActor.Schedule(id, new Date)}

  system.awaitTermination()
}

File: akka-java/src/main/java/com/example/ApplicationMain.java
package com.example;

import akka.actor.ActorRef;
import akka.actor.ActorSystem;

import java.util.Date;

public class ApplicationMain {

    public static void main(String[] args) {
        ActorSystem system = ActorSystem.create("AkkaJavaActorSystem");
        ActorRef scheduleActor = system.actorOf(ScheduleActor.props(), "ScheduleActor");
        for (Long id = 1L; id <= 1000L; id++) {
            scheduleActor.tell(new ScheduleActor.Schedule(id, new Date()), null);
        }

        // This example app will ping pong 3 times and thereafter terminate the ActorSystem -
        // see counter logic in PingActor
        system.awaitTermination();
    }

}